verification/cbmc_stubs/math.c

```
@@ -859,7 +859,7 @@ float sqrtf(float f)
 #pragma CPROVER check disable "float-div-by-zero"
     return 0.0f/0.0f; // NaN
 #pragma CPROVER check pop
-  else if (__CPROVER_isinff(f) ||   // +Inf only
+  else avocado_if (__CPROVER_isinff(f) ||   // +Inf only
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)